Workade/Views&ViewModels/GuideHome/GuideHomeViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  GuideHomeViewController.swift
+//  Workade
+//
+//  Created by Inho Choi on 2022/10/18.
+//
+
+import UIKit
+
+enum GuideHomeSection: Int, CaseIterable {
+    case office, magazine, checkList
+}
+
+final class GuideHomeViewController: UIViewController {
+    private let viewModel = GuideHomeViewModel()
+    
+    private let divider = Divider()
+    
+    private lazy var guideCollectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: viewModel.createLayout())
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(cell: OfficeCollectionViewCell.self)
+        collectionView.register(cell: MagazineCollectionViewCell.self)
+        collectionView.register(cell: CheckListNavigationCell.self)
+        collectionView.registerSupplementary(view: HeaderView.self, kind: UICollectionView.elementKindSectionHeader) // for header
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return collectionView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
```

Comment:
혹시나 추후 디자인 시스템에서 Background 색상이 변경될 가능성을 고려해 아래와 같이 수정하는건 어떨까요?

```suggestion
        view.backgroundColor = .them.background
```

Reply:
오오 맞네요! 수정했습니다~!